### PR TITLE
Bump protobuf pin to 3.0.0a3.

### PR DESCRIPTION
**Pull request by tseaver**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
     'google-apitools',
     'httplib2 >= 0.9.1',
     'oauth2client >= 1.4.6',
-    'protobuf == 3.0.0-alpha-1',
+    'protobuf == 3.0.0a3',
```
Comment (dhermes) [inline setup.py]: Can we go `>=`? Any reason not to?

File: tox.ini [hunk 2]
```diff
@@ -10,7 +10,7 @@ commands =
 deps =
     nose
     unittest2
-    protobuf==3.0.0-alpha-1
+    protobuf==3.0.0a3
```
Comment (dhermes) [inline tox.ini]: Ditto here with `>=`

File: setup.py (hunk 1 above)
Comment (tseaver) [inline setup.py]: a66ce1d

File: tox.ini (hunk 2 above)
Comment (tseaver) [inline tox.ini]: a66ce1d

**Comment by dhermes** [comment]
LGTM

This is one of those cases where two commits for such a tiny change just makes me crunch.

Squash into one if you have the energy.

**Pull request merged**